frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -297,7 +304,7 @@ export function Table({
               />
             );
           }
-          return <span>{cellValue}</span>;
+          return <span style={cellStyles}>{cellValue}</span>;
```

Comment:
Avoid inline-styles

Reply:
This is for customising the text colors, so they cannot be predefined as they are computed as per the configuration of the app.